src/common/memory_tracking.hpp

```diff
@@ -288,6 +288,8 @@ enum {
     key_nested_multiple,
     key_dw_conv_buffer,
     key_dw_conv_padded_bias,
+    key_decompression_scales,
```

Comment:
Is the comment above still valid?
```
// These two keys should always be the last ones,
// even though they are not in alphabetical order
```

Reply:
Not sure actually. But anyway moved new instances upper in the lise